docs/index.md

```diff
@@ -35,59 +35,42 @@ features:
 
 Welcome to all new visitors!
 
-This project is still very new and the team is still working hard on a number
-of areas of the whole project. That includes the website, package repository and
-GitHub organisation as well as the editor itself.
-
-The below is just some items that we are currently working on that might answer
-any questions you have at this early stage of the project.
-
 ### Downloads and Releases
 
 We are constantly working on the editor and maintain both a rolling release and regular release model for our binaries.
 You can find out more about the difference and download a binary of Pulsar by following [this link](https://pulsar-edit.dev/download.html)
 and following the instructions.
 
-We are looking to support distro repositories such as Debian/Ubuntu repos,
-Fedora, Snap, Flatpak, Arch/AUR etc. but we need to get to a good point with
-a stable release before we instigate that so for now the downloadable binaries
-are the main supported downloads.
+There are some community-maintained Pulsar packages in various formats, such as Flatpak, AUR, etc. The effort put into maintaining these packages is very much appreciated!
+That said, the core Pulsar team doesn’t have the person-hours or familiarity with the technology to be deeply involved with these efforts.
+The Pulsar Flatpak in particular has some minor known issues due to its sandboxing, but should still work overall.
+The core team will make some effort to support users on these packaged versions of Pulsar and answer their questions, but we have less control over these packages/environments,
+and less time to dedicate to them, so resolving issues may take more time and involve the community packaging maintainers, not just the core Pulsar team.
+For significant technical barriers like Flatpak sandboxing, related issues may have no estimated time of completion for a fix.
+Users are welcome to choose the packaging format they prefer, but your mileage may vary.
+
+### App Updates
+
+Currently Pulsar does not support automatic app updates. There are, however, automated notifications of new versions,
+which include links so you can easily get the new versions when they come out. (These notifications can be disabled if you wish.)
```

Comment:
```suggestion
which include links and instructions so you can easily get the new versions when they come out. (These notifications can be disabled if you wish.)
```
`pulsar-update` includes recognition of repo based installs as well (deb-get, flatpak, chocolatey) and shows instructions of what command to run.